op-node/rollup/derive/deposit_source.go

```diff
@@ -13,9 +13,10 @@ type UserDepositSource struct {
 }
 
 const (
-	UserDepositSourceDomain    = 0
-	L1InfoDepositSourceDomain  = 1
-	UpgradeDepositSourceDomain = 2
+	UserDepositSourceDomain       = 0
+	L1InfoDepositSourceDomain     = 1
+	UpgradeDepositSourceDomain    = 2
+	AfterForceIncludeSourceDomain = 3
```

Comment:
Maybe I am missing something but I do not see this in the specs: https://github.com/ethereum-optimism/specs/blob/main/specs/interop/derivation.md

Reply:
https://github.com/ethereum-optimism/specs/pull/370 closes this